Aptoide/aptoide-client-v8
# [bugfix/ASV-1854] - Glide transforms fix (& ASV-1851)

**What does this PR do?**

   This PR fixes an issue where certain Glide transforms weren't caching to memory/disk correctly. This will be noticeable once we enable transitions. It also removes unused methods. Additionally, this fixes a leaking chain (ASV-1851) since it was such a small change.

**Database changed?**

   No

**How should this be manually tested?**

  Honestly this is pretty hard to test, since it's only noticeable having transitions on (because if it isn't cached to memory, the fade-in will appear on scroll). So maybe just check if everything is ok with loading images.

**What are the relevant tickets?**

  [ASV-1854](https://aptoide.atlassian.net/browse/ASV-1854)
  [ASV-1851](https://aptoide.atlassian.net/browse/ASV-1851)

**Code Review Checklist**
